test/katello_test_helper.rb

```diff
@@ -98,6 +98,12 @@ module FixtureTestCase
 
     @@admin = ::User.unscoped.find(FIXTURES['users']['admin']['id'])
     User.current = @@admin
+
+    Organization.all.each do |org|
```

Comment:
For my tests I needed the organizations to have their labels set - which they don't because those fixtures are defined in Foreman and we relied on `get_organization` to do that for us. I tried to come up with a better solution which ended up requiring some VCRs to be regenerated. I'm not exactly sure *why* but the results look sane to me.